src/tools/run-make-support/src/lib.rs

```diff
@@ -332,6 +332,18 @@ pub fn read_dir<F: Fn(&Path)>(dir: impl AsRef<Path>, callback: F) {
     }
 }
 
+/// Check that `text` is equal to `expected`. Panic otherwise.
+#[track_caller]
+pub fn assert_equals(text: &str, expected: &str) {
+    if text != expected {
```

Comment:
Suggestion: maybe `expected` vs `actual` terminology here? (also update the doc comment)